pages/using_pulsar.md

```diff
@@ -0,0 +1,34 @@
+---
+layout: default
+title: Using Pulsar
+permalink: /using-pulsar/
+redirect_from:
+  - /using_pulsar.html
+sitemap:
+    priority: 0.7
+    lastmod: 2019-10-30T00:00:00-00:00
+---
+
+# <i class="fa fa-envelope"></i> Using Pulsar
+
+## Features
+
+[Pulsar](http://pulsar.apache.org/) is a popular publish-subscribe messaging system that supports both streaming and queueing use-cases. JHipster has an optional support for Pulsar, that will:
+
+- Configure [Pulsar clients](https://pulsar.apache.org/docs/2.11.x/client-libraries-java/) with JHipster.
+- Generate a Docker Compose configuration file, so Pulsar is usable by typing `docker-compose -f src/main/docker/pulsar.yml up -d`.
+- Generate an integration test using the [Spring Cloud Stream](https://docs.spring.io/spring-cloud-stream/docs/current/reference/html/) Pulsar binder and [Testcontainers](https://www.testcontainers.org/).
+
+## Prerequisite
+
+Generate a new application and make sure to select `Asynchronous messages using Apache Pulsar` when prompted for technologies you would like to use. A Docker Compose configuration file is generated and you can start Pulsar with the command:
+
+`docker-compose -f src/main/docker/pulsar.yml up -d`
+
+## Usage
+
+Check the [Spring for Apache Pulsar documentation](https://docs.spring.io/spring-pulsar/docs/0.2.0/reference/html/) for how to use Pulsar in JHipster.
```

Comment:
The versioned link should be outdated quickly and we use dependabot auto update, I think it's better to use current and add the versioned documentation at README.md.
```suggestion
Check the [Spring for Apache Pulsar documentation](https://docs.spring.io/spring-pulsar/docs/current/reference/html/) (prefer the versioned pulsar documentation link generated at your application's README.md) for how to use Pulsar in JHipster.
```